data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+			choice
+				`	"Who made this vault, what should I expect?"`
+				`	(No time for talking.)`
+					goto outside
+			`	"That is an information I cannot trust anyone with, not even you, and especially not here. I do not want the owner of this cache to be in danger whilst we need jump drives to expand. Assume it is owned by unfettered humans, or pirates as I think you call them, and that we are ready to face them," he tells you. He is fully armoured, meaning as long as you stay with him, everything should be fine.`
+			
+			label outside
+			`	Before exiting, he warns you, "Whatever happens, do not try anything stupid, and stay behind me at all times." He seems worried and keeps looking around, without making a sound. Well as little sound as possible, anyway, given he is in quite an impressive suit. It must have been crafted specifically for him in order to be so discrete. He does not have his helmet on, probably in order to see his surroundings better.`
+			choice
+				`	"Is everything alright?"`
+				`	"There is no need to be worried, that cache is well hidden."`
+				`	(Keep walking, ready to reach for my gun.)`
+					goto discrete
+			`	You are not sure what kind of sign he just did, but it seems pretty clear he wants you to keep quiet. Looking around, you notice the two of you have attracted quite a bit of attention from the locals. While they all seem to have nothing to lose, the vast majority of them are very young and probably not the hardened criminals you had expected.`
+			label discrete
+			`	As you get closer to the cache, more and more you get the impression that it is no simple, small depot. You pass several security turrets and assume that even more defenses are hidden. Eventually you notice a door, apparently leading to the underground vault. Suddenly, the Hai stops.`
+			choice
+				`	(Stay behind him.)`
+					goto stay
+				`	(Go to the door.)`
+			`	As you reach for the door the Hai shouts something, probably trying to warn you, but it is too late. You have already triggered the security systems, and get ripped to shreds by the turrets.`
+				die
+			
+			label stay
+			`	The Hai reaches for some kind of device in his pocket and enters what you assume to be the access codes. By the look on his face, it does not seem to be working very well, though... "Someone else has had access to the vault. This was not expected," he hisses as his grip tightens on his weapon and he dons his helmet. "Stay here, I will take care of it."`
+			choice
+				`	"I will not cower away whilst you do all the heavy lifting. I am coming with you."`
+					goto fight
+				`	(Do as he says.)`
+			`	It may have felt like the right thing to do at the time, but you quickly realize splitting up and then staying here in the open was not very intelligent. It is too late unfortunately, because you get stabbed by a local pirate, who is no longer impressed by your now missing armored friend.`
```

Comment:
This feels like it should be an "ask a question" scenario, not a "break everything" scenario.

Reply:
I wanted to convey the fact that this by no means a safe mission, but maybe I overdid it?
I could add some question options with these two staying the "let's get it done quickly" way?

Reply:
As a general rule, if the player is going to have the choice to do something obviously stupid, they should be able to ask a question with an answer that may encourage it, and a question with an answer that should make it obvious it's stupid.